app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
```

Comment:
- [ ] Kindly add the foreign key for all resources, this will help identify the author of either resource

Reply:
Hi @AshakaE , 

If you check my `db/migrate/schema.rb`, you will notice I used `user_id` instead of `author_id`. I also added them using migrations. The resources know their author `user_id`.

Thank you

Reply:
## Optional
yes, you are right! good job! but you can just mention it
like 
has_many :posts, foreign_key: 'user_id'
has_many :comments, foreign_key: 'user_id'
`has_many :likes, foreign_key: 'user_id'`

Reply:
Thank you for the feedback. I appreciate